apps/src/applab/PhoneFrame.jsx

```diff
@@ -73,39 +73,43 @@ const PhoneFrame = React.createClass({
     const { isDark, screenIds, showSelector, isPaused, onScreenCreate } = this.props;
     return (
       <span id="phoneFrame">
-        <div
-          style={[
-            styles.phoneFrame,
-            styles.phoneFrameTop,
-            isDark && styles.phoneFrameDark
-          ]}
-        >
-          {showSelector &&
-          <div style={styles.screenSelector}>
-            <ScreenSelector
-              screenIds={screenIds}
-              onCreate={onScreenCreate}
-            />
+        <div id="phoneFrameWrapper">
+          <div
+            id="phoneTop"
```

Comment:
I added PhoneFrameWrapper to scale down all the phone graphics on the edit screen.